.github/workflows/__test-workflow-docker-build-images.yml

```diff
@@ -164,9 +164,10 @@ jobs:
             const expectedTaggedVersions = isSigned ? 2 : 1;
 
             // Expected untagged versions:
-            // - For single platform: 0 (no multiarch manifest created)
-            // - For multi platform: number of platforms (one per platform)
-            const expectedUntaggedVersions = isSinglePlatform ? 0 : platforms.length;
+            // - For single platform: 1 (the digest-only push from build step)
```

Comment:
For single platform unsinged we must not have any untagged tags.

Proof of work: 

- Workflow: https://github.com/hoverkraft-tech/test-docker-build-images/actions/runs/18799787855
- Package versions: (only one for tag) https://github.com/hoverkraft-tech/test-docker-build-images/pkgs/container/test-docker-build-images%2Ftest-single-arch-unsigned/versions